src/components/counters/Counter.tsx

```diff
@@ -1,15 +1,66 @@
 import * as React from 'react';
 import cx from 'classnames';
-import Text from '../text/Text';
+import Text, {TEXT_COLOR} from '../text/Text';
 import Flex from '../flex/Flex';
 import Icon from '../icons/Icon';
 import type {IconTypeType} from '../icons/Icon';
 
 type CounterSizeType = 'xs' | 'xxs';
-type ColorType = 'red-60' | 'blue-60';
-export const COUNTER_COLOR = {
-  'red-60': 'red-60',
-  'blue-60': 'blue-60',
+
+export type CounterColorType =
+  | 'blue'
+  | 'green'
+  | 'indigo'
+  | 'red'
+  | 'yellow'
+  | 'gray'
+  | 'achromatic';
+
+export type CounterType = 'solid' | 'light';
+
+export const COUNTER_TYPE = {
+  solid: 'solid',
+  light: 'light',
+} as const;
+
+export const COLORS_SOLID_MAP = {
+  blue: 'blue-60',
+  green: 'green-60',
+  indigo: 'indigo-60',
+  red: 'red-60',
+  yellow: 'yellow-40',
+  gray: 'gray-40',
+  achromatic: 'black',
+} as const;
+
+const SOLID_COLOR_TEXT_MAP = {
+  blue: 'text-white',
+  green: 'text-white',
+  indigo: 'text-white',
+  red: 'text-white',
+  yellow: 'text-black',
+  gray: 'text-black',
+  achromatic: 'text-white',
+} as const;
+
+export const COLORS_LIGHT_MAP = {
+  blue: 'blue-20',
+  green: 'green-20',
+  indigo: 'indigo-20',
+  red: 'red-20',
+  yellow: 'yellow-20',
+  gray: 'gray-20',
+  achromatic: 'white',
+} as const;
+
+export const COUNTER_COLORS_SET = {
```

Comment:
Names `COUNTER_COLORS_SET` and `CounterColorType` should be somehow related.

Maybe just `COUNTER_COLORS` and `CounterColorType`. Prefix (COUNTER/Counter) is also optional but if you decided to use it it's also fine.